src/coreclr/jit/lsra.cpp

```diff
@@ -1284,14 +1291,18 @@ void LinearScan::doLinearScan()
     assert(blockEpoch == compiler->GetCurBasicBlockEpoch());
 
 #if TRACK_LSRA_STATS
-    if ((JitConfig.DisplayLsraStats() != 0)
+    if ((JitConfig.DisplayLsraStats() == 1)
 #ifdef DEBUG
         || VERBOSE
 #endif
         )
     {
         dumpLsraStats(jitstdout);
     }
+    else if ((JitConfig.DisplayLsraStats() == 2))
```

Comment:
nit: don't need double parens

Reply:
I actually forgot to remove this. We don't need `dumpLsraStatsCsv()` here.